hw/formal/fpv_all

```diff
@@ -18,34 +18,29 @@ done
 # list all blocks
 #-------------------------------------------------------------------------
 declare -a blocks=(
+  "alert_handler"
+  "aes"
   "gpio"
+  "flash_ctrl"
+  "hmac"
+  "otp_ctrl"
+  "pwrmgr"
   "rv_core_ibex"
   "rv_dm"
-  "spi_device"
   "rv_timer"
+  "spi_device"
   "uart"
-  "hmac"
-  "flash_ctrl"
-  "otp_ctrl"
-  "pwrmgr"
-  "usbuart"
   "usbdev"
-  "usb_fs_nb_pe"
```

Comment:
what happened to this guy? does this currently not compile?

Reply:
This one has two assertions and they are imported into usbdev. It does not have a FPV tb either. So I think testing it in usbdev is enough.

Reply:
ah ok then this should be fine, yes.

Reply:
Thanks michael.